schema-admin.graphql

```diff
@@ -487,6 +487,97 @@ input CreateApprovedCuratedCorpusItemInput {
     scheduledSurfaceGuid: ID
 }
 
+"""
+Input data for loading an Approved Item via an automated process and optionally scheduling
+this item to appear on a Scheduled Surface.
+"""
+input LoadApprovedCuratedCorpusItemInput {
```

Comment:
we need a `source` field as input  - can add them as enum 

may be `BACKFILL` and `CLIENT`

Reply:
created a follow up ticket to figure out this use case: https://getpocket.atlassian.net/browse/INFRA-376